go.work

```diff
@@ -1,6 +1,6 @@
 go 1.21.1
 
 use (
-    ./explorer
-    ./integration
-)
\ No newline at end of file
+	./explorer
+	./integration
```

Comment:
We should add the `hermes` entry I believe:

```suggestion
	./explorer
	./hermes
	./integration
```

I'm not sure, maybe someone can clarify, but the `go.work` would only come handy when running the integration tests, to locate the local app modules. Is there any other reason? Because if that's the only case it might not even be needed, depending on how the integration tests are written. If none of the local app modules is imported within the tests would make the `go.work` file redundant.